packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -47,6 +47,10 @@ export class Tooltip extends React.Component {
     });
   }
 
+  componentDidUpdate() {
+    this.popper.setOptions(this.props);
```

Comment:
A few questions because I don't remember the inner workings of this component
1. Do all the props for this component apply to popperjs? Or do we need to worry about only passing down a subset?
2. Should there be a conditional in this function that `setOptions` is only called if there is a difference between certain props? Or should `setOptions` be called any time any prop changes?

Reply:
Yes all the props deal with popper in some way. It should't effect anything other than modifying layout/attributes/etc  on the popper tooltip if changed.

Reply:
At some point this will be a functional component, and this will be handled a little more elegantly.